core-web/libs/sdk/react/src/lib/hooks/useDotcmsPageContext.tsx

```diff
@@ -0,0 +1,15 @@
+import { useContext } from 'react';
+
+import { PageProviderContext } from '../components/PageProvider/PageProvider';
+import { PageContext } from '../contexts/PageContext';
+
+/**
+ * Hook to get the page context
+ *
+ * @category Hooks
+ * @export
+ * @return {*}
+ */
+export function useDotcmsPageContext() {
+    return useContext<PageProviderContext>(PageContext);
```

Comment:
here you can extract logic from the components (parsing, transforming data) and expose them here, only if needed

Reply:
I am not sure if I want to change the rest API response, I think is better to keep it untouched in the context because users are already familiar with it.